interfaces/cython/cantera/reaction.pyx

```diff
@@ -2242,6 +2322,54 @@ cdef class BlowersMaselReaction(Reaction):
             r.setRate(rate_._rate)
 
 
+cdef class TwoTempPlasmaReaction(Reaction):
+    """
+    A reaction which rate coefficient depends on both gas and electron temperature
+
+    An example for the definition of an `TwoTempPlasmaReaction` object is given as::
+        rxn = TwoTempPlasmaReaction(
+            equation="O2 + E <=> O2-",
+            rate={"A": 17283, "b": -3.1, "Ea_T": -5820088, "Ea_Te": 10808733},
+            kinetics=gas)
+    The YAML description corresponding to this reaction is::
+        equation: O2 + E <=> O2-
+        type: two-temperature-plasma
+        rate-constant: {A: 17283, b: -3.1, Ea_T: -700 K, Ea_Te: 1300 K}
```

Comment:
Again, the 'T' in the notation here is not ideal, as you can specify activation energies in units other than temperature.